k8s/regions/oregon-b/dev.yaml

```diff
@@ -0,0 +1,41 @@
+AWS_REGION: us-west-2
+AWS_RESOURCE_STACK: SUMO-dev
+K8S_NAMESPACE: sumo-dev
+SUMO_READINESS_FAILURE_THRESHOLD: 1
+SUMO_READINESS_INITIAL_DELAY_SECONDS: 5
+SUMO_READINESS_PERIOD_SECONDS: 5
+SUMO_READINESS_SUCCESS_THRESHOLD: 1
+SUMO_READINESS_TCP_SOCKET_PORT: 8000
+SUMO_READINESS_TIMEOUT_SECONDS: 5
+SUMO_APP_NAME: sumo-dev
+SUMO_CPU_LIMIT: 200m
+SUMO_CPU_REQUEST: 100m
+SUMO_ENV_ALLOWED_HOSTS: sumo-dev.frankfurt.moz.works,dev.sumo.moz.works
+SUMO_ENV_AXES_BEHIND_REVERSE_PROXY: True
+SUMO_ENV_CELERY_ALWAYS_EAGER: True
+SUMO_ENV_CSRF_COOKIE_SECURE: True
+SUMO_ENV_DEBUG: False
+SUMO_ENV_DOMAIN: localhost
+SUMO_ENV_ENABLE_WHITENOISE: True
+SUMO_ENV_ENGAGE_ROBOTS: False
+SUMO_ENV_ES_LIVE_INDEXING: True
+SUMO_ENV_ES_USE_SSL: True
+SUMO_ENV_HTTPS: True
+SUMO_ENV_MEDIA_URL: https://dev-cdn.sumo.mozilla.net/
+SUMO_ENV_PIPELINE_ENABLED: True
+SUMO_ENV_PORT: 8000
+SUMO_ENV_READ_ONLY: True
+SUMO_ENV_SESSION_COOKIE_SECURE: True
+SUMO_ENV_SITE_URL: http://sumo-dev.frankfurt.moz.works
+SUMO_ENV_STAGE: False
+SUMO_IMAGE_PULL_POLICY: IfNotPresent
+SUMO_IMAGE_TAG: full-latest
```

Comment:
using a mutable tag in combination with the IfNotPresent pull policy is likely resulting in an older, broken image being used on one of the nodes, causing the `./bin/run-prod.sh: 7: exec: gunicorn: not found` error you described in https://github.com/mozilla/kitsune/pull/3034#issuecomment-363203330